sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2/async_samples/sample_analyze_invoices_async.py

```diff
@@ -200,8 +200,8 @@ async def analyze_invoice_async():
             unit_price = item.value.get("UnitPrice")
             if unit_price:
                 print(
-                    "......Unit Price: {} has confidence: {}".format(
-                        unit_price.value, unit_price.confidence
+                    "......Unit Price: {}{} has confidence: {}".format(
+                        unit_price.value, unit_price.value.code, unit_price.confidence
```

Comment:
if `code` is not returned by the service will this return empty string or None?

Reply:
We would be returning None. Just fixed how we print this. Thanks for pointing that out!